src/index.tsx

```diff
@@ -107,12 +247,14 @@ const getDefaultBreadcrumb = ({
   location: Location;
   pathSection: string;
 }): BreadcrumbData => {
-  const match = matchPath(pathSection, {
-    ...DEFAULT_MATCH_OPTIONS,
-    path: pathSection,
-  }) /* istanbul ignore next: this is hard to mock in jest :( */ || {
-    url: 'not-found',
-  };
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const match = matchPath(
+    {
+      end: true,
+      path: pathSection,
+    },
+    pathSection,
+  )!;
```

Comment:
I'm having trouble understanding why this is negated (`!`). Mainly because of the name `const match` I would assume it would be the other way around. I'm sure there's a good reason though, could you elaborate and maybe leave a comment?